### PR TITLE
Fix issue with checksum line format on Debian Wheezy base images.

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

**Comment by petrosagg** [comment]
I don't understand what this PR is fixing. Where do we use `%` as a field separator?

**Comment by nghiant2710** [comment]
@petrosagg, the issue here was `sha256sum` on wheezy is an old version and it only accepts checksum line with this format: `<checksum><space><space><filename>` (later versions accept both single or two spaces).
The checksum strings have 2 spaces https://github.com/resin-io-library/base-images/blob/master/debian/generate-dockerfile.sh#L13-L18 however they are trimmed so we need to set the field separator to something else to preserve two spaces.

File: debian/generate-dockerfile.sh [hunk 1]
```diff
@@ -75,7 +75,9 @@ for arch in $archs; do
 
 	# Tini
 	tiniBinary="tini$TINI_VERSION.linux-$arch.tar.gz"
+	IFS='%'
 	tiniChecksum="TINI_$arch" && tiniChecksum=$(eval echo \$$tiniChecksum)
```
Comment (petrosagg) [inline debian/generate-dockerfile.sh]: Setting the IFS is very hacky. You can instead do `tiniChecksum=$(eval echo \"\$$tiniChecksum\")`. Also I don't understand why you don't do `tiniChecksum=$(eval echo \"\$TINI_$arch\")` directly

Comment (nghiant2710) [inline debian/generate-dockerfile.sh]: updated!

**Pull request merged**